takwimu/templates/takwimu/_includes/navbar/about.html

```diff
@@ -4,69 +4,71 @@
 {% load datastructuretags %}
 
 {% block dropdown_id %}nav-about{% endblock %}
-
+<div class="text-right pr-3"></div>
 {% block dropdown_title %}About Takwimu{% endblock %}
 {% block dropdown_desc %}Takwimu provides quality insights and data-driven analysis to empower African development champions.{% endblock %}
 
 {% block dropdown_content %}
 <div class="row pl-4 pb-5" style="width: 100%;">
   <div class="col-sm-4">
-    <ul style="color:#7ab1f2; font-weight: bold;">
+    <ul style="color:#7ab1f2; font-weight: bold; line-height:2;">
       <li class="pb-2" type="disc"><a href="{% url 'about_page' %}"><u>About Takwimu</u></a></li>
-      <li class="pb-2" type="disc"><a href="/about/support-services"><u>Support Services</u></a></li>
+      <li class="pb-2" type="disc"><a href="/about/support-services"><u>Our Data Services</u></a></li>
+      <li class="pb-2" type="disc"><a href="https://medium.com/takwimu-africa"><u>Insights &amp; Stories</u></a></li>
       <li class="pb-2" type="disc"><a href="/contact-us"><u>Contact Us</u></a></li>
       <li class="mb-0 pb-3" type="disc"><a href="/dashboard"><u>Dashboard (Admin)</u></a></li>
```

Comment:
We should probably apply the underline to all the links in the dropdowns as well.